docker/ci-otio/Dockerfile

```diff
@@ -9,9 +9,10 @@ ARG ASWF_PKG_ORG=aswftesting
 ARG CI_COMMON_VERSION=2
 ARG VFXPLATFORM_VERSION=2021
 
-FROM ${ASWF_PKG_ORG}/ci-package-python:${VFXPLATFORM_VERSION} as ci-package-python
-FROM ${ASWF_PKG_ORG}/ci-package-qt:${VFXPLATFORM_VERSION} as ci-package-qt
-FROM ${ASWF_PKG_ORG}/ci-package-pyside:${VFXPLATFORM_VERSION} as ci-package-pyside
+# We are currently not interested in the Python implementation and bindings
```

Comment:
Not sure if Python is really not needed. Need to investigate whether something runs the Python interpreter or if there's always some Python supported built into the library.

Reply:
Adapters are all Python-based, and there is no real plan to port them to C++. We don't use adapters, however.